GAMES.json

```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260":
+	{
+		"Comment": "Requires Mono"
```

Comment:
What @directhex said, as well as we do not wish to collect that information. It is all in the contribution section of the README, what will be accepted and what wont.